packages/mobx-state-tree/src/utils.ts

```diff
@@ -483,3 +483,30 @@ export function setImmediateWithFallback(fn: (...args: any[]) => void) {
         setTimeout(fn, 1)
     }
 }
+
+/**
```

Comment:
Add experimental note, that this might change on minor releases

Reply:
Done